subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/buildcache/TaskOutputCachingJavaPerformanceTest.groovy

```diff
@@ -310,26 +257,22 @@ class TaskOutputCachingJavaPerformanceTest extends AbstractTaskOutputCacheJavaPe
         println "Changed file dates in $count cache artifacts in $dir in ${time} ms"
     }
 
-    private String getRemoteCacheSettingsScript() {
-        """
-            def httpCacheClass = Class.forName('org.gradle.caching.http.HttpBuildCache')
-            buildCache {
-                local {
-                    directory = '${cacheDir.absoluteFile.toURI()}'
-                }
-                remote(httpCacheClass) {
-                    url = '${buildCacheServer.uri}/' 
-                    push = ${pushToRemote}
-                }
-            }
-        """.stripIndent()
-    }
-
     private def setupTestProject(JavaTestProject testProject, String tasks) {
         runner.testProject = testProject
         runner.gradleOpts = ["-Xms${testProject.daemonMemory}", "-Xmx${testProject.daemonMemory}"]
         runner.tasksToRun = tasks.split(' ') as List
         runner.cleanTasks = ["clean"]
     }
 
+    /**
+     * In order to compare the different cache backends we define the scenarios for the
+     * tests here.
+     */
```

Comment:
This should be a regular comment, not Javadoc. It also doesn't really give more context to me. How's this help compare different backends?

Reply:
I removed the comment.